.github/workflows/copilot-setup-steps.yml

```
@@ -12,29 +12,88 @@ jobs:
         run: |
-          dotnet build
+          dotnet build --no-restore
+        continue-on-error: true
+
+      - name: Test environment
+        run: |
+          dotnet test --no-build --verbosity minimal
         continue-on-error: true
 
       - name: Environment summary
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)